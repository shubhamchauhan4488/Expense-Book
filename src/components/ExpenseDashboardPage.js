import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';
import ExpensesSummary from './ExpensesSummary';
import ExpenseCharts from './ExpenseCharts'

const ExpenseDashboardPage = () => (
    <div>
        <ExpensesSummary />
        <ExpenseListFilters />
        <ExpenseList/>
        <ExpenseCharts/>
    </div>
)

export default ExpenseDashboardPage;