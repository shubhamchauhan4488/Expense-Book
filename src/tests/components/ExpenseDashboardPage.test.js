import React from "react";
import { shallow } from 'enzyme'
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage'

test("Should render Header correctly", () => {
    const wrapper = shallow(<ExpenseDashboardPage/>);
    expect(wrapper).toMatchSnapshot();
   //  expect(wrapper.find('h1').text()).toBe('Expensify');
   }
   )
