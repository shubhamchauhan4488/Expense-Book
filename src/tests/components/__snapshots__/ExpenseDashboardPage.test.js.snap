// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Header correctly 1`] = `
<div>
  This is from ExpenseDashboardPage
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
